src/logging.c

```diff
@@ -62,7 +62,8 @@ enum rcutils_colorized_output
 bool g_rcutils_logging_initialized = false;
 
 char g_rcutils_logging_output_format_string[RCUTILS_LOGGING_MAX_OUTPUT_FORMAT_LEN];
-static const char * g_rcutils_logging_default_output_format = "[{severity}] [{name}]: {message}";
+static const char * g_rcutils_logging_default_output_format =
+  "[{time}]: [{severity}] [{name}]: {message}";
```

Comment:
Looking at the ROS 1 style, would `[{severity}] [{time}] [{name}]: {message}` make more sense?